tuxemon/map.py

```diff
@@ -440,10 +440,19 @@ def __init__(
         self.inits = inits
         self.events = events
         self.renderer: Optional[pyscroll.BufferedRenderer] = None
-        self.edges = edges
+        self.edges = maps.get("edges")
         self.data = tiled_map
         self.sprite_layer = 2
         self.filename = filename
+        self.maps = maps
+
+        # optional fields
+        # inside (true), outside (none)
+        self.inside = maps.get("inside")
```

Comment:
Can you convert inside and town to books?

book(maps.get("inside"))

Reply:
no problem